<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1746</width>
    <height>1265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="view" name="view">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>1500</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>890</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CreateObjectPoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Perim and area</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>540</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Changing:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>settransformpoint</string>
    </property>
   </widget>
   <widget class="QPushButton" name="creation">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Creation parameters</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ASide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="changeA">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Bside</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="changeB">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MainX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="xspinbox">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>MainY</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="yspinbox">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_15">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>PolSide</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="recSide">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Triangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rectangle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_11">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pentagon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ellipse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_13">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Hexagon</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>380</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="font">
        <font>
         <family>Century Gothic</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Transformation:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButton_5">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Scale</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_4">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>89</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>-50</number>
         </property>
         <property name="maximum">
          <number>1500</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>-20</number>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>5</number>
         </property>
         <property name="displayIntegerBase">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>221</width>
      <height>250</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QPushButton" name="pushButton_6">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ChangeRec</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QSpinBox" name="recchA">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="recchB">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>ChangeEll</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QSpinBox" name="ellchA">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>501</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="ellchB">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="minimum">
            <number>5</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_13">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="pushButton_14">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ChangeTr</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_15">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ChangeHex</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_16">
           <property name="font">
            <font>
             <family>Century Gothic</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>ChangePent</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSpinBox" name="trchA">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <number>5</number>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>201</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Degrees:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="inputMethodHints">
          <set>Qt::ImhDigitsOnly</set>
         </property>
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Coefficient:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox">
         <property name="font">
          <font>
           <family>Century Gothic</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="minimum">
          <double>0.500000000000000</double>
         </property>
         <property name="maximum">
          <double>5.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1746</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>view</class>
   <extends>QGraphicsView</extends>
   <header location="global">view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
